boards/tenstorrent/tt_blackhole/support/tt_blackhole_bmc.cfg

```diff
@@ -5,14 +5,9 @@ transport select hla_swd
 source [find target/stm32g0x.cfg]
 # No reset signal is attached to the STM32 debug header
 reset_config none
+gdb_breakpoint_override hard
 
-$_TARGETNAME configure -event gdb-attach {
-        echo "Debugger attaching: halting execution"
-        reset halt
```

Comment:
This makes sense to me - normally, I wouldn't expect a gdb-attach to reset the target, it should just attach to it in whatever state it's in.

And I think the default detach behaviour is identical anyway.

No real need to echo a message in either case.